services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/docker_compose_specs.py

```diff
@@ -182,6 +182,19 @@ def _update_resource_limits_and_reservations(
         spec["environment"] = environment
 
 
+_METRICS_PREFIX: Final[str] = "io.simcore.container"
```

Comment:
I would add this is some common library so that we make sure all labels (of images, services, containers, etc) use **the same** [reverse domain-name notation](https://en.wikipedia.org/wiki/Reverse_domain_name_notation)

Somehow connect with the ideas in https://github.com/ITISFoundation/osparc-simcore/issues/3638